administrator/language/en-GB/lib_joomla.ini

```
@@ -614,8 +614,10 @@ JLIB_INSTALLER_ERROR_PACK_UNINSTALL_INVALID_MANIFEST="Package Uninstall: Invalid
 JLIB_INSTALLER_ERROR_PACK_UNINSTALL_INVALID_NOTFOUND_MANIFEST="Package Uninstall: Manifest file invalid or not found: %s"
 JLIB_INSTALLER_ERROR_PACK_UNINSTALL_LOAD_MANIFEST="Package Uninstall: Could not load manifest file."
 JLIB_INSTALLER_ERROR_PACK_UNINSTALL_MANIFEST_NOT_REMOVED="Package Uninstall: Errors were detected, manifest file not removed!"
+JLIB_INSTALLER_ERROR_PACK_UNINSTALL_MISSING_EXTENSION="Package Uninstall: This extension is missing or has already been uninstalled: %s"
 JLIB_INSTALLER_ERROR_PACK_UNINSTALL_MISSINGMANIFEST="Package Uninstall: Missing manifest file."
 JLIB_INSTALLER_ERROR_PACK_UNINSTALL_NOT_PROPER="Package Uninstall: This extension may have already been uninstalled or might not have been uninstalled properly: %s"
+; The following string is deprecated and will be removed with 6.0
```

please use the same comment as elsewhere

```suggestion
; @deprecated 5.1 will be removed in 6.0
```

---

Why 5.1 and not 4.4?

---

Or 4.4.2?

---

I can't find any `@deprecated` (with the `@` in any ini file), so I guess your example is from PHP. When looking for `deprecated` without the `@` I get:
```
administrator\language\en-GB\com_config.ini:COM_CONFIG_FIELD_LOG_CATEGORIES_DESC="A comma separated list of log categories to include or exclude. Common log categories include but are not limited to: database, databasequery, database-error, deprecated and jerror. If empty, custom logging is disabled."
administrator\language\en-GB\com_config.ini:COM_CONFIG_FIELD_LOG_EVERYTHING_DESC="Logs everything, except deprecated APIs."
administrator\language\en-GB\com_media.ini:; The string COM_MEDIA_OPEN_ITEM_ACTIONS is deprecated and no longer in use as of 4.2.4
administrator\language\en-GB\com_privacy.ini:COM_PRIVACY_CORE_CAPABILITY_LOGGING_IP_ADDRESS="Joomla's logging system records the IP address of the visitor which leads to a message being written to its log files. These log files are used to record various activity on a Joomla site, including information related to core updates, invalid login attempts, unhandled errors, and development information such as the use of deprecated APIs. The format of these log files may be customised by any extension which configures a logger, therefore you are encouraged to download and review the log files for your website which may be found at `%s`."
administrator\language\en-GB\com_workflow.ini:; All the following strings are deprecated and will be removed with 5.0
administrator\language\en-GB\lib_joomla.ini:JLIB_FORM_VALUE_CACHE_WINCACHE="Windows Cache (deprecated)"
administrator\language\en-GB\lib_joomla.ini:JLIB_FORM_VALUE_SESSION_WINCACHE="Windows Cache (deprecated)"
administrator\language\en-GB\lib_joomla.ini:JLIB_UTIL_ERROR_DOMIT="DommitDocument is deprecated. Use DomDocument instead."
administrator\language\en-GB\plg_system_debug.ini:; All the following strings are deprecated and will be removed with 5.0
administrator\language\en-GB\plg_system_debug.ini:PLG_DEBUG_LOGS_DEPRECATED_FOUND_TEXT="The code that is marked as deprecated will not work in upcoming Joomla versions, please review it."
administrator\language\en-GB\plg_system_debug.ini:PLG_DEBUG_LOGS_DEPRECATED_FOUND_TITLE="%s deprecated messages logged!"
language\en-GB\com_media.ini:; The string COM_MEDIA_OPEN_ITEM_ACTIONS is deprecated and no longer in use as of 4.2.4
language\en-GB\files_joomla.sys.ini:; All the following strings are deprecated and will be removed with 6.0
language\en-GB\lib_joomla.ini:JLIB_FORM_VALUE_CACHE_WINCACHE="Windows Cache (deprecated)"
language\en-GB\lib_joomla.ini:JLIB_FORM_VALUE_SESSION_WINCACHE="Windows Cache (deprecated)"
language\en-GB\lib_joomla.ini:JLIB_UTIL_ERROR_DOMIT="DommitDocument is deprecated. Use DomDocument instead."
```
So to me my comment seems to be ok.

---

sorry I was looking in the 5.0 codebase where the comments are updated